README.md

```diff
@@ -1 +1,41 @@
-# Hello-Microverse-Project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> Hello Microverse Text in H1 Element Tag.
+
+
+## Built With
+
+- HTML
+- CSS
+- Lighthouse
+- Webhint
+- Stylelint
+
+## Getting Started
+
+#### cloning project
+git clone https://github.com/collinsmezie/Hello-Microverse-Project
+
+## Getting packages and debuging with Stylelint npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x 
+
+
+##### For validation detection using Stylelint Run npx stylelint "**/*.{css,scss}" 
+
+##### from parent source directory 
+
+## Getting packages and debuging with Webhint npm init -y npminstall --save-dev hint@6.x 
+
+##### For validation detection using Webhint Run npx hint . 
+
+### Prerequisites
+- VsCode
+
+### Licence MIT License ##### from parent source directory Copyright (c) 2022 Sentayhu Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions: The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software. THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE. 
```

Comment:
- [ ] Great job on updating the readme file, however, I think  there is no need to add this agreement here, I would suggest you remove them from your readme file to make it more professional, remember, your readme should be filled in accordance with your project